x-pack/legacy/plugins/reporting/server/plugin.ts

```diff
@@ -40,7 +40,9 @@ export class ReportingPlugin
     // regsister setup internals
     this.reportingCore.pluginSetup({ browserDriverFactory, elasticsearch });
 
-    return {};
+    return {
+      reportingCore: this.reportingCore,
```

Comment:
FIXME: I should probably just pick the properties that are OK for outside plugins to consume.

Right now the one I think is useful is the `getScreenshotsObservable` function